### PR TITLE
feat(finanzonline): Add root and bootloader detection removal patch

### PR DESCRIPTION
**Pull request by 1fexd**
Removes the root and bootloader check from [FinanzOnline [+]](https://play.google.com/store/apps/details?id=at.gv.bmf.bmf2go)

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/finanzonline/detection/bootloader/fingerprints/BootloaderDetection2Fingerprint.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.finanzonline.detection.bootloader.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+
+object BootloaderDetection2Fingerprint : MethodFingerprint(
```
Comment (FineFindus) [inline src/main/kotlin/app/revanced/patches/finanzonline/detection/bootloader/fingerprints/BootloaderDetection2Fingerprint.kt:6]: That not a good name. Maybe rename it to `BootStateFingerprint`?

File: src/main/kotlin/app/revanced/patches/finanzonline/detection/bootloader/patch/BootloaderDetectionPatch.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.finanzonline.detection.bootloader.fingerprints.BootloaderDetection2Fingerprint
+import app.revanced.patches.finanzonline.detection.bootloader.fingerprints.BootloaderDetectionFingerprint
+
+
+@Patch
+@Name("remove-unlocked-bootloader-detection")
```
Comment (FineFindus) [inline src/main/kotlin/app/revanced/patches/finanzonline/detection/bootloader/patch/BootloaderDetectionPatch.kt:19]: Kind of verbose name. Could be simply shortened to `remove-bootloader-detection` or `bootloader-detection`.

**Comment by oSumAtrIX** [comment]
Why is this patch relevant? Does the app prevent being used if the device is rooted?

**Comment by 1fexd** [comment]
> 

Ye

> Why is this patch relevant? Does the app prevent being used if the device is rooted?

Yeah it does

**Pull request merged**